test/common.js

```diff
@@ -497,6 +497,12 @@ function fail(msg) {
 }
 exports.fail = fail;
 
+exports.mustNotCall = function(msg) {
+  return function dontCall() {
+    fail(msg || 'function should not have been called');
```

Comment:
Can we do something to prevent this from printing stuff like

```
AssertionError: [object Object]
```

when `msg` is not a string?